docs/pages/architecture/overview.mdx

```diff
@@ -234,11 +227,11 @@ valid for 12 hours by default, unless you specify another interval via the
 
 You can learn more about `tsh` in the [User Manual](../server-access/guides/tsh.mdx).
 
-### TCTL
+### `tctl`
```

Comment:
Should we add a paragraph here about authenticating?

I think it would be worth mentioning that:
- tctl can be run on the auth server directly, if the config file is specified.
- tctl can pick up a tsh profile automatically